content/en/tracing/guide/init_resource_calc.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Init container resource usage
```

Comment:
feel free to adjust description as needed but was mainly targeting the sentence capitalization here

```suggestion
title: Init Container Resource Usage
Learn how Datadog Agent v7.60+ automatically manages resource allocation for init containers that set up tracing, ensuring reliable tracer startup without impacting pod scheduling.
```